src/pipeline_v3/core/postgres_base_rls.py

```
@@ -0,0 +1,379 @@
+                min_size=self.settings.min_connections,
+                max_size=self.settings.max_connections,
+                timeout=30,
+                check=ConnectionPool.check_connection,
+            )
+            self._is_initialized = True
+            logger.info(
+                f"Initialized PostgreSQL connection pool for schema: {self.schema}"
+                f"{f' (tenant: {self.tenant_id})' if self.tenant_id else ' (admin mode)' if self.is_admin else ''}"
+            )
+        except Exception as e:
+            logger.error(f"Failed to initialize connection pool: {e}")
```

Ruff (0.11.9)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)